webapp/components/interop.js

```diff
@@ -1251,15 +1396,16 @@ class InteropFeatureChart extends PolymerElement {
         keepInBounds: true,
         maxZoomIn: 4.0,
       },
-      colors: ['#4285f4', '#ea4335', '#fbbc04'],
+      // Line chart color definitions for [Chrome, Firefox, Safari, Interop].
+      colors: ['#7be83f', '#F57400', '#0095F0', '#FCBA2F'],
     };
 
     // We draw the chart in two ways, depending on the viewport width. In
     // 'full' mode the legend is on the right and we limit the chart size to
     // 700px wide. In 'mobile' mode the legend is on the top and we use all the
     // space we can get for the chart.
     if (containerDiv.clientWidth >= 700) {
-      options.width = 700;
+      options.width = '100%';
```

Comment:
Is the comment described above still correct (we limit the chart size to
    // 700px wide)?  If not, maybe we should update the comment.

Reply:
updated for clarity